src/main/scala/uk/gov/hmrc/play/it/MicroServiceEmbeddedServer.scala

```diff
@@ -97,11 +101,10 @@ trait EmbeddedServiceOrchestrator extends ResourceProvider with StartAndStopServ
     val config: Configuration = play.api.Configuration.from(configMapUpdated)
 
 
-    val application = new DefaultApplication(new File("."), this.getClass.getClassLoader, None, applicationMode) {
-      override def configuration: Configuration = super.configuration ++ config
-    }
+    val application: Application = new GuiceApplicationBuilder(configuration = config).build()
```

Comment:
I don't think its relevant anymore, if you look in the Guice application builder without explicit configuration it simply defaults to an empty set.
It looks reasonable to assume that all the relevant config is that defined locally.